hathor/pubsub.py

```diff
@@ -62,7 +61,7 @@ class HathorEvents(Enum):
 
         CONSENSUS_TX_REMOVED:
             Triggered when a tx is removed because it became invalid (due to a reward lock check)
-            Publishes the tx hash
+            Publishes the tx object
```

Comment:
Should we emphasize that the metadata will have the values just before the removal? I'm wondering whether we should just skip the metadata information.

Reply:
I think it makes sense to remove the metadata in this case. I'll check with Abadesso if it would complicate things on the wallet service side.

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/1106/commits/9a42b1f050682e28b5bc61bb85ef55d9ab0f3f4f

Reply:
Great! Thanks!